src/patternfly/components/Button/button-icon.hbs

```diff
@@ -1,5 +1,4 @@
 <i class="fas{{#if button-icon--type}} fa-{{button-icon--type}}{{/if}}{{#if button-icon--modifier}} {{button-icon--modifier}}{{/if}}"
-  aria-hidden="true"
```

Comment:
This change isn't needed. `aria-hidden="true" should remain on the `<i>` element. This attribute hides the icon from screen readers, because the sr-accessible text is included on the `<button>` element instead using `aria-label`.